Modify installation guide for Windows and Getting-Started page (#523)
jamulussoftware/jamulussoftware.github.io: wiki/en/Getting-Started.md

## Installation

To get Jamulus working with your operating system, please follow the installation guide for your platform:

* [Windows](/wiki/Installation-for-Windows)
* [macOS](/wiki/Installation-for-Macintosh)
* [Linux](/wiki/Installation-for-Linux)

And for now, close all other applications. It’s best to start simple at first.

## Hardware Setup

> If you use external audio hardware, plug that in before you start Jamulus. If you haven't set it up, please see the installation guides mentioned above.

Reviewer: "If you haven't set it up" - what's the "it", your hardware or Jamulus?

> If you use external audio hardware, plug that in before you start Jamulus. If you haven't set it up, please see the installation guides mentioned above.

Author: ```suggestion
If you use external audio hardware, plug that in before you start Jamulus. If you haven't configured your hardware, please see the installation guides mentioned above.
```

Author: Maybe that’s the problem here:

you might need to configure your hardware (turn up gain,…) and set it up in JACK/ASIO4ALL/… see my proposal

Reviewer 2: Maybe better to add the thing about plugging in hardware to the platform-specific guides (which don't currently say this) and remove it here to avoid confusion. I think it's unlikely people with external hardware would not have it plugged in at this stage, and "configuration" is platform specific. This guide is for all platforms.

Reviewer 2: BTW I thought this issue was about the links on this page: https://github.com/jamulussoftware/jamuluswebsite/issues/488 If so, you may as well put them in as part of this PR.

Author: Adding it to every page means duplication. The main intention about that was that the most problems seem to occur with wrongly configured hardware.

Author: > BTW I thought this issue was about the links on this page: #488 If so, you may as well put them in as part of this PR.

Can do that.

Author: See: https://github.com/jamulussoftware/jamuluswebsite/pull/524 it might also go live right away (Fuzzy matching?)

Reviewer 2: > Adding it to every page means duplication

Not sure I understand. If we say  you should plug in your hardware before running Jamulus here, but don't say it in the section about installing Jamulus earlier, what is the benefit of saying it here for the first time?  Perhaps a minor thing, but seems a little confusing. 

On balance, I wouldn't say it at all since I highly doubt anyone would not have their hardware attached to their machine when investigating how to use Jamulus.

Reviewer 3: > See: #524 it might also go live right away (Fuzzy matching?)

Since they're links that don't need translation (would be copied from source segment), if you run the update and then create all target scripts, the new links will appear in translations without having to do anything. Translators could validate them eventually but it's not strictly necessary, other that it "artificially" lowers the % of translated text for threshold purposes.

Author: > what is the benefit of saying it here for the first time? 

My original intention here was to hint multiple user groups (somewhat experienced PC user without pro audio knowledge should get a hint at the fact that the gear plays a role; inexperienced user who is already overwhelmed by the fact that he needs to install software -> needs a step by step guide) at one of the probably most challenging UX problems: setting up the audio gear. If it was part of every OS setup page, we‘ll need to write the same text (Plug in Hardware/configure your internal sound card/…) everywhere. Maybe it’s more of related to the UX issue of Windows setup?

Maybe this needs some more general discussion/thoughts?

Reviewer 2: No that's OK, it's just that if we say "If you haven't configured your hardware... " then that leaves the user to ask themselves if they have configured their hardware or not, and if they think they haven't, they have to go to the installation page they probably just came from where we don't in fact tell them to plug in their hardware (not sure why, maybe because it's obvious?) and that has platform-specific instructions (well, for Windows and Linux anyway). Maybe it's just a minor thing.

> The most important thing about servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a server with a ping of less than 15ms if you can.

Reviewer 2: ```suggestion
The most important thing about servers is their “ping time”. The bigger the number, the harder it will be to play in time with others. Usually, you would select a server with a ping of less than 50ms if you can.
```

Increasing the reccommended ping time might encourage people to use more public servers, as @jujudusud points out.

Author: 50ms is quite high… 

30ms? However, it all depends on the hardware too. 

Why don’t we just say "marked green or yellow" in the connection dialog?

Reviewer 4: As discussed with pljones, 50ms is the real maximum we had used for a server.
We want to encourage users to connect to the existing public server.

Author: Ok. So applied that and merged PR.
